### PR TITLE
FEATURE: YAML configuration file support [ISSUE-89]

### PR DESCRIPTION
**Pull request by ivanilves**
ISSUE: https://github.com/ivanilves/lstags/issues/89

## WHA?
We need to enter the magic world of hipsters and hype, but how could we do it without YAML support? :wink: 

## Next step?
Kubernetes chart for `lstags`

# GIF

### CODE REVIEW

**Comment by ivanilves** [comment]
Maybe we need more reviews, and not only from @vonrabbe who is the :zap: merger? :trollface: 

@solidnerd 
@zmalik 
@Fsero

File: docker/config/config_test.go [hunk 1]
```diff
@@ -73,6 +70,8 @@ func TestLoad(t *testing.T) {
 }
 
 func TestLoadWithIrrelevantConfigFile(t *testing.T) {
+	const irrelevantConfigFile = "../../fixtures/docker/config.json.irrelevant"
```
Comment (zmalik) [inline docker/config/config_test.go]: could be just `irrelevantConfigFile := "../../fixtures/docker/config.json.irrelevant"` right?

File: util/fix/fix.go [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package fix
+
+import (
+	"os"
+	"strings"
+)
+
+// Path resolves "~" intro the real home path
+func Path(path string) string {
+	return strings.Replace(path, "~", os.Getenv("HOME"), 1)
```
Comment (zmalik) [inline util/fix/fix.go]: @ivanilves check the golang package ` os/user` you can get home directory there, maybe more elegant

File: docker/config/config_test.go (hunk 1 above)
Comment (ivanilves) [reply docker/config/config_test.go]: yes, `const` here is confusing, right?

File: util/fix/fix.go (hunk 2 above)
Comment (ivanilves) [reply util/fix/fix.go]: good observation!

**Comment by ivanilves** [comment]
@vonrabbe let's merge!

**Pull request merged**